### DISCUSSION TITLE
3rd Party Integrations

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3673] [Team Posts]
@jekkos It appears that several people are asking for 3rd party integrations for eCommerce RESTful API integration not all that unlike what I'm doing with CLCdesq (See @webshells website tickets... there are about a half dozen of them).  PR #2619 needs to get merged into the master before anyone can develop these integrations.  Can you take a look at my comments on that PR and help me sort out just how you want this Event Generator to work?  Perhaps it's because the discussion is happening over Github, but I'm having a difficult time understanding just what you want changed and how to go about that.

**Comment by jekkos** [comment]
OK I'll  have a further look somewhere this week hopefull. I'm currnently preparing to move house so I might not have too much time to spend on the project beside this coming two weeks.

**Comment by objecttothis** [comment]
Understood.  Happy moving!

**Comment by objecttothis** [comment]
@jekkos, how did the move go?

**Comment by jekkos** [comment]
Good been rather busy ofcourse the last two weeks. I'll try to look into this somewhere tonight if that's ok?

**Comment by objecttothis** [comment]
yeah, that's fine.  I'm glad that the move went well.
